src/mudlet.cpp

```diff
@@ -1210,6 +1210,46 @@ void mudlet::commitLayoutUpdates()
     }
 }
 
+bool mudlet::setFontSize(Host* pHost, const QString& name, int size)
```

Comment:
💭 Do we want to clamp the font size settings to a range of 1 to 39 inclusive? If so we ought to check and error out on it in the Lua caller - so that it can send a __nil+error message__ and display the duff value as well as the valid ones...